### ISSUE TITLE
[Bar Charts] tooltip should look similar to line chart

### ISSUE THREAD

**Issue by sstone2423** [carbon-design-system/carbon-charts#657]
#### I am submitting a...

- [x] Feature request
- [x] Design defect?

### Issue description

Bar chart tooltips only should the value with no dataset label or color. This can be confusing and difficult to read as the user needs more context to figure out which dataset is belongs to, Additionally, this is an accessibility issue for color blind users as you have to then correspond the colored bar you're highlighting with the legend to determine which dataset it is.

#### Steps to produce the issue

1. Go to https://carbon-design-system.github.io/carbon-charts/react/?path=/story/bar-vertical--simple-bar-discrete
2. Highlight a bar
3. See the issue

#### Current behavior

Only shows the value in the tooltip

#### Expected behavior

Shows the dataset name and color in the tooltip

### Screenshot or recording

![Screen Shot 2020-06-05 at 2 24 31 PM](https://user-images.githubusercontent.com/25177649/83915148-4b351a80-a738-11ea-98ed-ceae1610c10d.png)

![Screen Shot 2020-06-05 at 2 20 36 PM](https://user-images.githubusercontent.com/25177649/83915095-2fca0f80-a738-11ea-90f6-ec3cfd496afa.png)

**Comment by theiliad** [comment]
the behaviour you're currently seeing is what's in the spec. I think that that makes sense for all bar types except for stacked.

stacked should show the multiline tooltip onHover on the grid, however on the bars the current behaviour still makes sense

**Comment by sstone2423** [comment]
Stacked would definitely make sense to have the multiline. I made a workaround to replicate that single bar state for the time-being with the `tooltip.customHTML` but its pretty hacky

**Comment by theiliad** [comment]
Tooltips have been redesigned in #550, although the new specs haven't been developed.

You're able to completely customize tooltips btw:
```js
{
  tooltip: {
   customHTML: (data, defaultHTML) => "return html string"
  }
}
```

**Comment by theiliad** [comment]
Closed by #695

**Comment by darq37** [comment]
There's still a problem of not being able to set up a custom component as a tooltip.
